src/legacy/core_plugins/vis_default_editor/public/default_editor.tsx

```diff
@@ -33,67 +29,46 @@ import { getInitialWidth } from './editor_size';
 
 function DefaultEditor({
   embeddable,
-  savedObj,
+  vis,
   uiState,
```

Comment:
Seems props `uiState` & `embeddable` became redundant. Should be removed from `editor.render` function passing down and clean up the `EditorRenderProps`